### PR TITLE
remove confusing ci step

### PR DESCRIPTION
**Pull request by OnnoEbbens**

### CODE REVIEW

**Comment by OnnoEbbens** [comment]
Failing tets not related to this PR

**Review by martinvonk — APPROVED** [review]
Great job!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -44,16 +44,8 @@ jobs:
...
-        if: ${{ github.event_name == 'push' }}
-        env:
-            NHI_GWO_USERNAME: ${{ secrets.NHI_GWO_USERNAME}}
-            NHI_GWO_PASSWORD: ${{ secrets.NHI_GWO_PASSWORD}}
-        run: |
-          py.test ./tests -m "not notebooks"
-
       - name: Run tests only
-        if: ${{ github.event_name == 'pull_request' }}
+        if: ${{ github.event_name == 'pull_request' || github.event_name == 'push'}}
```
Comment (bdestombe) [inline .github/workflows/ci.yml:48]: The if statement can be removed. The workflow is triggered on pr and push, so which other events do you exclude with this line?

File: nlmod/dims/grid.py [hunk 2]
```diff
@@ -237,6 +237,12 @@ def modelgrid_from_ds(ds, rotated=True, nlay=None, top=None, botm=None, **kwargs
 
 def modelgrid_to_vertex_ds(mg, ds, nodata=-1):
     """Add information about the calculation-grid to a model dataset."""
+
+    logger.warning(
```
Comment (bdestombe) [inline nlmod/dims/grid.py:241]: Deprecation warning instead?

**Review by bdestombe — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (OnnoEbbens) [reply .github/workflows/ci.yml]: good point!

**Comment by OnnoEbbens** [comment]
again: failing test not related to this PR

**Pull request merged**